assets/js/modules/ads/index.js

```diff
@@ -75,6 +76,9 @@ export function registerModule( modules ) {
 				'google-site-kit'
 			),
 		],
+		SettingsDisconnectNoteComponent: isFeatureEnabled( 'adsPax' )
```

Comment:
This is not enough to just check for the feature flag enabled. The idea for this note is to make sure that users understand that their PAX campaigns will continue even if they disable this module. So, we need to show this notice if the PAX setup flow has been used.